src/DeckOfCards.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * Created by sparatan117 on 11/28/16.
+ */
+// stores a deck of cards, vlaues of the cards, face of card, and tells if it is black.
+public class DeckOfCards {
```

Comment:
This class does not contain what the name of the card is so like Jack, Queen and King.  Their value might all be 10 but they are not the same card.